templates/var/lib/matrix-synapse/conf.d/email.yaml.j2

```diff
@@ -20,11 +20,11 @@ email:
 #   #
 #   notif_for_new_users: true
 #
-#   # Defining a custom URL for Riot is only needed if email notifications
-#   # should contain links to a self-hosted installation of Riot; when set
+#   # Defining a custom URL for Element is only needed if email notifications
+#   # should contain links to a self-hosted installation of Eiot; when set
```

Comment:
Seems like a typing error. Should be "Element" instead of "Eiot"?

Reply:
That's right! That was half done